src/css/base.css

```diff
@@ -21,6 +21,13 @@
 
 html {
   scroll-behavior: smooth;
+  scroll-padding-top: 100px;
+}
+
+@media screen and (min-width: 1280px) {
+  html {
+    scroll-padding-top: 170px;

```

Comment:
Why `170px`? Lest keep `100px` or some value in range 100-150

Reply:
170px;
<img width="1437" alt="image" src="https://github.com/olexandrd/goit-markup-team-project/assets/13751198/2d94b200-f02e-43e7-b12d-7a44453d396b">

Reply:
120px;
<img width="1433" alt="image" src="https://github.com/olexandrd/goit-markup-team-project/assets/13751198/fa03500e-6c8b-4845-862e-eede39abaf1b">

Reply:
made 100px for all sizes